packages/connect-node/src/node-universal-client.ts

```diff
@@ -437,10 +437,10 @@ type Sentinel = Promise<void> & {
   isResolved(): boolean;
 
   /**
-   * Reject the sentinel. All errors are converted to ConnectError via
-   * ConnectError.from().
+   * Reject the sentinel. Unless the reason is already a ConnectError, it is
+   * converted to one via ConnectError.from().
    */
-  reject: (reason: ConnectError | unknown) => void;
+  reject: (reason: unknown) => void;
```

Comment:
`ConnectError | unknown` is redundant because `unknown` is greedy, and the effective type is `unknown`. We have been using the redundant typing for documentation, but the new version of the linter disallows it.

Updated the types to not be redundant, and clarified in the JSDoc where applicable instead.